Gemfile

```diff
@@ -47,6 +47,9 @@ group :test do
   gem 'paperclip', ['>= 3.4', '!= 4.3.0']
   gem 'poltergeist', '~> 1.5'
   gem 'rack-cache', require: 'rack/cache'
+  gem 'refile', '>= 0.5.5', require: 'refile/rails'
+  gem 'refile-mini_magick', '>= 0.1.0'
+  gem 'refile-mongoid', '>= 0.0.1'
```

Comment:
`refile-mongoid` should go into `:mongoid` group, since it is only required for mongoid builds.